public_dropin_notebook_environments/python311_notebook/env_info.json

```
@@ -1,10 +1,10 @@
 {
   "id": "6583d56f5627082b3cff990e",
-  "name": "[DataRobot] Python 3.11 Notebook Drop-In",
-  "description": "This template environment can be used to create Python 3.11 notebook environments.",
+  "name": "(deprecated)[DataRobot] Python 3.11 Notebook Drop-In",
+  "description": "Deprecated in favour of Python 3.11 Base Notebook Drop-In Template",
```

Nit:
I think (per docs team) we use American English spelling.
```suggestion
  "description": "Deprecated in favor of Python 3.11 Base Notebook Drop-In Template",
```